### PR TITLE
Adding french translation for message bundle

### PR DESCRIPTION
**Pull request by jbleduigou**
Signed-off-by: Jean-Baptiste Le Duigou <jb.leduigou@gmail.com>

See [JENKINS-61489](https://issues.jenkins-ci.org/browse/JENKINS-61489).

@fcojfernandez

### CODE REVIEW

**Comment by fcojfernandez** [comment]
thanks for this PR @jbleduigou !

This week I don't have availability to review it, but I will have a look next one

**Comment by fcojfernandez** [comment]
@alecharp @batmat could you help me with this review? thanks in advance, mates!

File: src/main/resources/com/cloudbees/hudson/plugins/folder/Messages_fr.properties [hunk 1]
```diff
@@ -0,0 +1,12 @@
+AbstractFolder.BeingDeleted={0} est en cours de suppression
+AbstractFolder.FailureToStopBuilds=Impossible de suspendre et de stopper {0,choice,1#{0,number,integer} construction|1<{0,number,\
+  integer} constructions} de {1}
+Folder.DisplayName=Dossier
+Folder.DefaultPronoun=\u00c9l\u00e9ment
+Folder.Description=Cr\u00e9e un conteneur qui stocke des objets imbriqu\u00e9s. Utile pour grouper ensemble des \u00e9l\u00e9ments. \
+   Contrairement \u00e0 une vue qui n''est qu''un filtre, un dossier cr\u00e9e un espace de nommage distinct, de sorte que vous \
+  pouvez avoir plusieurs \u00e9l\u00e9ments du m\u00eame nom tant qu''ils se trouvent dans des dossiers diff\u00e9rents.
+
+RelocateAction.permission.desc=N\u00e9cessaire pour d\u00e9placer un job d''un dossier (ou de la racine Jenkins) \u00e0 un autre.
```
Comment (alecharp) [inline src/main/resources/com/cloudbees/hudson/plugins/folder/Messages_fr.properties]: maybe `job` -> `projet`.

```suggestion
RelocateAction.permission.desc=N\u00e9cessaire pour d\u00e9placer un projet d''un dossier (ou de la racine Jenkins) \u00e0 un autre.
```

Comment (jbleduigou) [reply src/main/resources/com/cloudbees/hudson/plugins/folder/Messages_fr.properties]: Thanks for the feedback!
I have updated my changes accordingly.

**Comment by fcojfernandez** [comment]
thanks @alecharp !

Merging since the error in the CI is not related to this PR: a problem with the memo in the EC2 linux agent happened when the deploy of the incremental stage was executed

**Pull request merged**